examples/clone/clone.bal

```diff
@@ -35,29 +35,29 @@ public function main() {
         address: address
     };
 
-    // Assign the `person` value to an `any` typed variable called `anyValue`.
+    // Assigns the `person` value to an `any`-typed variable called `anyValue`.
     // The type of the value held by `anyValue` is determined at runtime.
     any anyValue = person;
 
-    // Invoke the `.clone()` built-in method. The return type is either `Person` or `error`. Note that `error` could be
-    // returned if the variable on which `.clone()` is called is not of type `anydata`.
+    // Invokes the `.clone()` built-in method. The return type is either `Person` or `error`. `error` could be
+    // returned if the variable on which `.clone()` is called is not of the type `anydata`.
     // If the type of the value that is cloned can be determined at compile time and is `anydata`, the type of the
     // return value is exactly the type of the value being cloned.
-    // e.g. `Person result = person.clone();`
+    // E.g., `Person result = person.clone();`
```

Comment:
```suggestion
    // e.g., `Person result = person.clone();`
```

Reply:
Capital "E" should be used as this is the beginning of a new sentence.